services/dqa-secondary-reports/main.py

```diff
@@ -127,11 +127,16 @@ def update_repo(self):
 
         logger.debug('done updating repo')
 
-    def serve_field_totals(self, version):
+    def serve_field_totals(self, version, table=None, field=None):
         "Entrypoint for Flask routing for a given version of PEDSnet data model"
         def versioned_data():
             try:
-                info = self.field_totals[version]
+                if table is None:
+                    info = self.field_totals[version]
+                elif field is None:
+                    info = self.field_totals[version][table]
+                else:
+                    info = self.field_totals[version][table][field]
             except HTTPError:
```

Comment:
This is definitely late in the review, but none of these operations would raise an `HTTPError` since that is specific to `requests`. They could raise `KeyError`, but I believe Flask already catches that and raises a 500. So.. you can remove the `try/except`.